### PR TITLE
development screening and assessment

### PR DESCRIPTION
**Pull request by maulidkategile**

### CODE REVIEW

File: opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java [hunk 1]
```diff
@@ -383,4 +385,23 @@ protected void evaluateExclusiveBreastFeeding(Map<String, ServiceWrapper> servic
...
+
+        Alert alert = serviceWrapper.getAlert();
+        if (alert == null || new LocalDate().isBefore(new LocalDate(alert.startDate()))) return;
+
+        Map<String, List<VisitDetail>> details = getDetails(Constants.Events.DEVELOPMENT_SCREENING_AND_ASSESSMENT);
+
+        BaseAncHomeVisitAction action = new BaseAncHomeVisitAction.Builder(context, MessageFormat.format(context.getString(R.string.pnc_child_development_screening_assessment), ""))
+                .withOptional(false)
+                .withDetails(details)
+                .withProcessingMode(BaseAncHomeVisitAction.ProcessingMode.SEPARATE)
```
Comment (gosso22) [inline opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java]: Can you confirm the processing mode here is separate?

**Review by gosso22 — COMMENTED** [review]
Can you check the processing mode on the ChildInteractor, confirm if this is the standard approach. When you specify the processing mode to be separate, it means that this will be treated as its own event rather than Child Home Visit

**Comment by issyzac** [comment]
@maulidkategile work to resolve the conflics so that this can be merged

File: opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/PncHomeVisitInteractorFlv.java [hunk 2]
```diff
@@ -127,8 +128,10 @@ private void evaluateOtherActions() throws Exception {
             evaluateExclusiveBreastFeeding(baby);
             evaluateNutritionStatusBaby(baby);
             evaluateObsIllnessBaby(baby);
-            evaluatePlayAssessmentCounseling(baby);
             evaluateSkinToSkin(baby);
+            evaluateSkinToSkin(baby);
```
Comment (issyzac) [inline opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/PncHomeVisitInteractorFlv.java]: This looks like a duplicate

**Review by issyzac — APPROVED** [review]
LGTM

**Pull request merged**